src/gazebo_mavlink_interface.cpp

```diff
@@ -205,6 +205,11 @@ void GazeboMavlinkInterface::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf
 
   getSdfParam<std::string>(_sdf, "motorSpeedCommandPubTopic", motor_velocity_reference_pub_topic_,
       motor_velocity_reference_pub_topic_);
+  if (_sdf->HasElement("actuatorOutputsCommandPubTopic")) {
+    getSdfParam<std::string>(_sdf, "actuatorOutputsCommandPubTopic", actuator_outputs_pub_topic_,
+    actuator_outputs_pub_topic_);
+    pub_actuator_outputs_ = true;
+  }
```

Comment:
Why do we need a separate topic to command actuator outputs?

Reply:
I believe I added this thinking that later with multiple agents we can run different gazebo topics - but maybe that is already done with namespacing?

Reply:
Yeap you multivehicle simulation is already supported